examples/automm/kaggle_california_house_price/example_kaggle_house.py

```diff
@@ -33,10 +33,8 @@ def get_parser():
 
 def get_automm_hyperparameters(mode, text_backbone, cat_as_text):
     if mode == "ft-transformer":
-        hparams = {"model.names": ["categorical_transformer",
-                                   "numerical_transformer",
-                                   "hf_text",
-                                   "fusion_transformer"],
+        hparams = {"model.names": ["ft_transformer",
+                                   "hf_text"],
```

Comment:
Shouldn't we keep `fusion_transformer` to fuse `ft_transformer ` and `hf_text`?

Reply:
Yes, will update